README.md

```diff
@@ -67,9 +67,21 @@ to revise it in response.
 * Build consensus and integrate feedback. RFCs that have broad support
 are much more likely to make progress than those that don't receive any
 comments.
-* Eventually, somebody on the [core team] will either accept the RFC by
-merging the pull request and assigning the RFC a number, at which point
-the RFC is 'active', or reject it by closing the pull request.
+* Eventually, the [core team] will decide whether the RFC is a candidate
```

Comment:
We discussed at the core team meeting that the details of how subteams work hasn't yet been through the RFC process, and therefore we can't yet make them a formal part of the process. That's coming soon, though.